tests/testthat/test-calcOutput.R

```diff
@@ -269,6 +283,27 @@ test_that("Aggregation works", {
   })
 })
 
+test_that("Bilateral aggregation works", {
+  calcBilateral <- function() {
+    map <- toolGetMapping("regionmappingH12.csv")
+    tmp <- expand.grid(map[[2]], map[[2]], stringsAsFactors = FALSE)
```

Comment:
Is there a reason why stringsAsFactors is set explicitly here but not in calcOutput (line 125)?
```suggestion
    expandedMapping <- expand.grid(map[[2]], map[[2]], stringsAsFactors = FALSE)
```

Reply:
could catch...I will add it there as well